### DISCUSSION TITLE
Add tablet rotate changing to Osu!Lazer android

### DISCUSSION THREAD

**Discussion by MegaTKC** [ppy/osu#23246] [Ideas]
I'm able to get my Star G640s working with Osu!Lazer on my Pixel 6a, however in the game, my tablet is stuck at 90 degrees, and I'd like to change it to 0 degrees.

**Comment by ItsShamed** [comment]
It seems that there is no support yet for tablets on Android. The fact that you can get your tablet to work is half surprising. #16494
